src/read.rs

```diff
@@ -59,9 +59,11 @@ where
 {
     // tell writer about our epoch tracker
     let epoch = sync::Arc::new(atomic::AtomicUsize::new(0));
+
```

Comment:
Did `rustfmt` introduce this newline? If not, I'd prefer to keep it the way it was.

Reply:
Indeed, it was rustfmt. I have it set up to automatically format on file save.

Reply:
How weird -- I didn't expect it to force an extra line here. In that case, I'll resolve all of these!